### PR TITLE
cat pixels and combine_sqw

### PR DESCRIPTION
**Pull request by abuts**
Depends on #1373 beeing merged.

This is partial solution of #1330 concerning cat pix and combine_sqw. Though combine sqw is not fully Page_op type operation, it would be difficult to implement if in other way then two-stage operation, combining pixels in big blob on the first stage.

This is what is done here.

### CODE REVIEW

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

File: horace_core/sqw/page_operations/PageOp_cat_pix.m [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                npix = varargin{i}.num_pixels;
+                obj.npix(i) = npix ;
+                npix_tot_ = npix_tot_ + npix;
+            end
+            obj.npix_tot = npix_tot_;
+        end
+        function is = get_exp_modified(obj)
+            % if sqw object is processed, here we want to know unique
+            % run_id
+            is = ~isempty(obj.img_);
```
[the remark this answers is not in the document]
Comment (abuts) [reply horace_core/sqw/page_operations/PageOp_cat_pix.m:142]: sqw object has image and pix does not.

File: horace_core/configuration/@planner_config/planner_config.m [hunk 2]
```diff
@@ -185,7 +185,7 @@
             fields = this.saved_properties_list_;
         end
         %
-        function value = get_internal_field(this,field_name)
+        function value = get_default_value(this,field_name)
```
[the remark this answers is not in the document]
Comment (abuts) [reply horace_core/configuration/@planner_config/planner_config.m:188]: here in particular, it is always default

File: herbert_core/configuration/@config_base/config_base.m [hunk 3]
```diff
@@ -74,16 +91,19 @@
...
+        % function value = get_default_value(obj,field_name)
+        %     % method gets default property value ignoring current value, stored
+        %     % in configuration and returned by usual get.property method
+        %     %
+        %     % Default protected field names, corresponding to property names
+        %     % normaly nave the form:
+        %     % protected_prop_name = [public_prop_name,'_']
+        %     % so default implementation of this function have the fome:
+        %     %
+        %     value = obj.([field_name,'_']);
```
[the remark this answers is not in the document]
Comment (abuts) [reply herbert_core/configuration/@config_base/config_base.m:104]: see previous comment or `config_base` description.

**Comment by abuts** [comment]
merging as further work depends on this. I hope I've addressed the reviewer comments

**Pull request merged**